releasenotes/notes/43709.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: istioctl
+releaseNotes:
+  - |
```

Comment:
Can we get the issue noted here as well (https://github.com/istio/istio/issues/43691).

Reply:
Also, RMs should looking at release notes for anything cherry-picked since that tells them something was changed instead of a new feature which may not necessarily have a RN.

Reply:
I see, thanks